CPP/Question_19/README.md

```diff
@@ -0,0 +1,25 @@
+
+Question:
+
+C++ program to displaying numbers using function
+
+
+
+
+
+

```

Comment:
These many line breaks is unnecessary in a markdown file. One line break in between each content is more than enough.